TECHMANIA/Assets/Scripts/Paths.cs

```diff
@@ -64,52 +69,81 @@ public static void PrepareFolders()
     #region Things in working directory
     public static string GetTrackRootFolder()
     {
+// Normalize Windows platform slash to \
```

Comment:
Since only Application.streamingAssetsPath is causing the slash problem, it is unnecessary to replace slashes here.

Reply:
It still necessary or the go up button will cause error when click go up button in game folder.
`KeyNotFoundException: The given key was not present in the dictionary.`

Reply:
`currentLocation` logs in `OnGoUpButtonClick` is `C:/Users/Kento/AppData/LocalLow/DJ Hitori/TECHMANIA\Tracks\sub`

Reply:
> `currentLocation` logs in `OnGoUpButtonClick` is `C:/Users/Kento/AppData/LocalLow/DJ Hitori/TECHMANIA\Tracks\sub`

Looks like your build target is iOS, but I see keeping the replace will make debugging easier, so let's keep it.